### PR TITLE
fix content posts

### PR DESCRIPTION
**Pull request by cantsin**
- renames blog posts to reflect 18f.gsa.gov's current date
- removes stray divs from "hacking bureaucracy" post
- regenerates cache

### CODE REVIEW

File: content/posts/2017-09-28-three-ways-to-manage-research-projects.md
Comment (beechnut) [inline content/posts/2017-09-28-three-ways-to-manage-research-projects.md:1]: clarification: What's happening here? The filenames look identical but the names don't line up perfectly. Is it a dash encoding thing?

Screenshot 2024-08-15 at 12 15 58 PM

File: content/posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.md
Comment (beechnut) [inline content/posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.md:1]: clarification: Would you explain what's happening here?

I thought it was HTML beautification at first but it looks like you removed... is it a hardcoded SVG?

**Review by beechnut — APPROVED** [review]
Approved! But if you can answer the clarifying questions, that'll satisfy my burning curiosity

File: content/posts/2017-09-28-three-ways-to-manage-research-projects.md
Comment (cantsin) [reply content/posts/2017-09-28-three-ways-to-manage-research-projects.md:1]: there's a space in the filename 😞  `-to `

File: content/posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.md
Comment (cantsin) [reply content/posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.md:1]: For some reason the "next" and "previous" blog posts were hard coded here (in the blog post!), including the SVG for the left and right arrows. (This must have been an earlier version of the 18f.gov.site). 

Even that I would have left alone but the tags didn't even close properly, which messed up the rest of the page, including the social media links, footer, etc etc. I removed everything except the actual content (the speakerdeck) which is what really matters here.

**Pull request merged**